pet.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+from subprocess import *
+import os
+import signal
+import click
+
+
+cli = click.Group()
+
+
+@cli.command()
+@click.argument('name')
+def start(name):
+    """starts new project"""
+    if os.path.exists("./projects/%s" % name):
+        if not os.path.isfile("./projects/%s/_lock" % name):
+            open("./projects/%s/_lock" % name, 'a')
```

Comment:
here in start it would be best to use context manager for entering shell (adding lock) and exiting shell (removing lock).

Example of context manager:
```
class File():

    def __init__(self, filename, mode):
        self.filename = filename
        self.mode = mode

    def __enter__(self):
        self.open_file = open(self.filename, self.mode)
        return self.open_file

    def __exit__(self, *args):
        self.open_file.close()
```